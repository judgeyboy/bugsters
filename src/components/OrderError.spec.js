import { shallow } from 'vue-test-utils'

import OrderError from './OrderError'

describe('OrderError Component', () => {

  it('should be a Vue instance', () => {
    const wrapper = shallow(OrderError)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(OrderError)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
