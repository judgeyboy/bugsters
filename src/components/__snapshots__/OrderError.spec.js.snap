// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderError Component should match snapshot 1`] = `
"<div class=\\"order-error\\"><h2 class=\\"text-center\\">Error :(</h2> <p>
    Oh well this is embarrassing! It looks like
    something went horribly wrong...please try again
  </p></div>"
`;
